### PR TITLE
Add BlockedClientMeasureTime to coordinator

### PR DESCRIPTION
**Pull request by ashtul**
Added `RedisModule_BlockedClientMeasureTimeStart` and `RedisModule_BlockedClientMeasureTimeEnd` to get the thread time when a command run has the client is blocked, passed to a thread and the command call ends on the main thread.

Fixes https://github.com/RediSearch/RediSearch/issues/2686

### CODE REVIEW

**Comment by ashtul** [comment]
protect against older Redis versions

File: src/concurrent_ctx.c [hunk 1]
```diff
@@ -80,7 +80,9 @@ static void threadHandleCommand(void *p) {
   if (!(ctx->options & CMDCTX_KEEP_RCTX)) {
     RedisModule_FreeThreadSafeContext(ctx->ctx);
   }
-
+  if (ctx->options & CMDCTX_COORD) {
```
Comment (MeirShpilraien) [inline src/concurrent_ctx.c:83]: Can you explain why this flag is neede?

File: src/concurrent_ctx.c [hunk 2]
```diff
@@ -105,7 +107,9 @@ int ConcurrentSearch_HandleRedisCommandEx(int poolType, int options, ConcurrentC
   for (int i = 0; i < argc; i++) {
     cmdCtx->argv[i] = RedisModule_CreateStringFromString(cmdCtx->ctx, argv[i]);
   }
-
+  if (options & CMDCTX_COORD) {
```
Comment (MeirShpilraien) [inline src/concurrent_ctx.c:110]: Same question

**Review by MeirShpilraien — COMMENTED** [review]
:+1: Small questio

File: src/concurrent_ctx.c (hunk 1 above)
Comment (ashtul) [reply src/concurrent_ctx.c]: Got mixed in the callbacks.
Removed

**Review by MeirShpilraien — APPROVED** [review]

**Review by MeirShpilraien — DISMISSED** [review]

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**